daos-cortx/docs/setup_robinhood_policy_engine.md

```diff
@@ -0,0 +1,149 @@
+
+`/root/setup_robinhood/robinhood-3.1.6/scripts/rbh-config create_db <db_name>    'localhost' 'rbh_password'`
+
+A common name for robinhood database name is 'rbh_fsname'. Write the selected password to a file only readable by 'root' (600), for example in /etc/robinhood.d/.dbpassword.
+
+Reference is available [here.](https://github.com/cea-hpc/robinhood/wiki/v3_posix_tuto#configuration)
+
+* Create a robinhood configuration file, starting with a simple robinhood template:
+
+`cp /root/setup_robinhood/robinhood-3.1.6/doc/templates/basic.conf /etc/robihood.d/posix.conf`
+
+* Edit the configuration file
+
+In 'General' block, set filesystem root path, and the corresponding file system type.
+       
+       fs_path = "/fs/root";
+       fs_type = xfs;
+ 
+* In 'ListManager' block, set database connection parameters:
+
+      db = rbh_fsname;
+      password_file = "/etc/robinhood.d/.dbpassword" ;
+
+It is recommended to define your fileclasses before running the initial file system scan.
+
+This way, you will get relevant information in 'rbh-report --class-info' report after the initial scan is completed.
+
+    fileclass empty_file {
+        definition { type == file and size == 0 }
+     }
+     fileclass small_file {
+        definition { type == file
+                 and size > 0
+                 and size <= 32MB }
+     }
+
+Reference config file is available [here.](https://github.com/Seagate/cortx-experiments/blob/main/daos-cortx/src/samples/posix.conf)
+Make sure to make above relevant changes in fields mentioned above.
+
+* Start Robinhood scan and update
+
+ `/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/robinhood --scan --once -L stderr -f /etc/robinhood.d/posix.conf`
+
+* Query database
+ 
+ `/root/setup_robinhood/robinhood-3.1.6/rpms/BUILD/robinhood-3.1.6/src/robinhood/rbh-report --fs-info --class-info`
+ 
+# Test for daos to CORTX and vice-versa data movement
+
+In this exercise object movements from daos to cortx and cortx to daos will be made sure. This exercise will be carried out by defining policies inside a container and running those policies using robinhood policy engine. There are two prerequisites to carry out this test mentioned below.
+
+* Daos server node running on one VM
+
+  - follow this [document](https://github.com/Seagate/cortx-experiments/blob/main/daos-cortx/docs/setup_daos.md) to setup daos and creating container.
+  
+* Cortx server node running on second VM
+
+  - follow this [document](https://github.com/Seagate/cortx/blob/main/QUICK_START.md) to setup cortx on your vm.
+  
+* Once everything is in-place let's start with the data movement test by following the steps below on daos node which is hosting robinhood policy engine.
+ 
+1. Create objects in a daos container (i.e. inside dfuse mount point)
+
+- create larger and smaller objects inside the container. container is mounted at /mnt/daos_container for this exercise and same fs_path will be used in config file as well.
```

Comment:
this can be simple text blocks and not bullets.

Reply:
done